libs/core/src/schemas/commands/community.schemas.ts

```diff
@@ -80,6 +80,36 @@ export const SetCommunityStake = {
   output: Community,
 };
 
+export const CreateStakeTransaction = {
```

Comment:
Are you writing a policy that invokes this command when reacting to new transactions, or is this a projection of the transactions?

Reply:
Our currently implementation the client just makes the crypto transaction through metamask (Doesn't go through our backend).

My intention with this API is that it will be called by the client directly after they make the crypto transaction.

Reply:
Yeah that is correct. Basically this is a bandaid solution until Tim implements the full system which will read through the events on chain and then project them into our database.

Reply:
@Rotorsoft as @kurtisassad said this is a 'flaky' route which is to be expected. _Most_ events will be recorded and the gaps will be filled by EVM CE 2.0 per #6947. So the system will be eventually consistent and not permanently inconsistent.